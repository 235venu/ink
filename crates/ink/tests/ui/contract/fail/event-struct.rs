#[ink::event_definition]
pub struct Event {}

fn main() {}
