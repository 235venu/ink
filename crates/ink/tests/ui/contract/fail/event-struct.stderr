error: expected `enum`
 --> tests/ui/contract/fail/event-struct.rs:2:5
  |
2 | pub struct Event {}
  |     ^^^^^^

error: ink! event definitions must be enums
 --> tests/ui/contract/fail/event-struct.rs:1:1
  |
1 | #[ink::event_definition]
  | ^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = note: this error originates in the attribute macro `ink::event_definition` (in Nightly builds, run with -Z macro-backtrace for more info)
